src/shared/components/comment/comment-nodes.tsx

```diff
@@ -28,10 +32,20 @@ export class CommentNodes extends Component<CommentNodesProps, any> {
   }
 
   render() {
-    let maxComments = this.props.maxCommentsShown ?? this.props.nodes.length;
+    const maxComments = this.props.maxCommentsShown ?? this.props.nodes.length;
 
-    return (
-      <div className="comments">
+    const borderColor = this.props.depth
+      ? colorList[this.props.depth % colorList.length]
+      : colorList[0];
+
+    return this.props.nodes.length > 0 ? (
+      <ul
```

Comment:
Should this be an ordered list, since it's sorted by some criterion?

Reply:
I could make it ordered. Whichever you think would be better for accessibility. However, I don't think children of top level comments get sorted, so that's worth keeping in mind.

Reply:
As @fastfinge  said, ul makes more sense from a screen reader perspective. With an ol, you'd (depending on settings) get the item number and the number of items on the list: the comment number and the total number of comments at that depth, in that thread. That seems like a lot of extra info to process.